crates/torin/src/values/size.rs

```diff
@@ -267,35 +271,88 @@ impl<'a> DynamicCalculationEvaluator<'a> {
         Some(lhs)
     }
 
-    /// Parse and evaluate the value with the following grammar:
+    /// Parse and evaluate the term, implements implicit multiplication. only parenthesis count as
+    /// a seperator, so syntax like 50 50 isnt correct, but 50(50) is because the parenthesis act
+    /// as a seperator
+    fn parse_term(&mut self) -> Option<f32> {
+        let prefix = self.parse_prefix()?;
+        let mut lhs = None;
+        // set to true so that the first value is multiplied and counts as normal syntax
+        let mut last_is_seperator = true;
+
+        while let Some((rhs, seperator)) = self.parse_value() {
+            if last_is_seperator || seperator {
+                lhs = Some(lhs.unwrap_or(1.0) * rhs);
+            } else {
+                return None;
+            }
+            last_is_seperator = seperator;
+        }
+        if let Some(prefix) = prefix {
+            match prefix {
+                DynamicCalculation::Add => lhs,
+                DynamicCalculation::Sub => lhs.map(|v| v * -1.0),
+                _ => unreachable!(),
```

Comment:
Maybe put some message in the `unreachable!()`, although it would be better if this was integrated into the parsing error reporting instead of panicking, but this is better done in another PR

Reply:
Added something there now